pkg/runctl/controller_test.go

```diff
@@ -355,7 +355,7 @@ func Test_Controller_syncHandler_OnTimeout(t *testing.T) {
 
 	assert.Assert(t, status != nil)
 	assert.Equal(t, api.StateFinished, status.State)
-	assert.Equal(t, status.State, status.StateDetails.State)
+	assert.Equal(t, api.StateCleaning, status.StateDetails.State)
```

Comment:
I don't like that State is finished and StateDetails is cleaning.
I think StateDetails should always be the the same as current State - which is finished in this case.
We should add StateDetails finished with the same date for startedAt and FinishedAt.